README.adoc

```diff
@@ -127,6 +127,19 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
```

Comment:
_Configurations_ still does not mean anything to me. For now, I'd just skip this level. Let's pull 'HTTP Proxy Settings' up to the level of 'Configurations' and remove it.